analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LinkedListPropertiesInsteadOfMethods.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+class MyClass
+{
+    void Various()
```

Comment:
Looks too long and should be split up. The name of the method already implies that it deals with too many scenarios.